spec/models/shopify_import/custom_collection_spec.rb

```diff
@@ -0,0 +1,29 @@
+require 'spec_helper'
+
+RSpec.describe ShopifyImport::CustomCollection, type: :model do
+  subject { described_class.new }
+
+  before do
+    Spree::Config[:shopify_api_key] = 'api_key'
+    Spree::Config[:shopify_password] = 'password'
+    Spree::Config[:shopify_shop_domain] = 'shop_domain.myshopify.com'
+  end
+
+  it 'is inheriting from base' do
```

Comment:
This spec really doesn't test any functionality. Having `.count` and `.all` specs covers the fact that this class responds to such methods _and_ that the functionality of such methods is correct. If it acts like a duck and it quacks like a duck, it is duck: https://stackoverflow.com/a/4205163